sdk/src/main/java/tech/relaycorp/sdk/RelaynetClient.kt

```diff
@@ -1,22 +1,36 @@
 package tech.relaycorp.sdk
 
-import tech.relaycorp.sdk.endpoints.ExternalEndpoints
-import tech.relaycorp.sdk.endpoints.InternalEndpoints
-import tech.relaycorp.sdk.gateway.Gateway
-import tech.relaycorp.sdk.messaging.Messaging
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.asFlow
+import kotlinx.coroutines.flow.flatMapConcat
 
 object RelaynetClient {
```

Comment:
I'm not sure this object adds much in this new version. The only need we need from it is to get a unique instance of the client, which could well be a static method on the client class (probably the most common way to implement the singleton pattern anyway).

I'm also concerned that it proxies all the methods on the gateway. I'm not sure what it buys the app developer, and it's going to be a maintenance overhead as those methods evolve (they'd have to be kept in sync).

Reply:
Alternatively, could/should this object be just an instance of `GatewayClient`?